api-get-policy.go

```diff
@@ -79,7 +80,7 @@ func (c Client) getBucketPolicy(bucketName string) (policy.BucketAccessPolicy, e
 	urlValues.Set("policy", "")
 
 	// Execute GET on bucket to list objects.
-	resp, err := c.executeMethod("GET", requestMetadata{
+	resp, err := c.executeMethod(context.Background(), "GET", requestMetadata{
```

Comment:
@poornas, is there any reason why we are not accepting letting developers their own context here?

Reply:
@vadmeste, the scope of this PR is to limit adding context to PutObject calls because it is likely to be long running enough to warrant a request cancellation. There is not much of a use case for GetBucketPolicy to use context.

Reply:
:+1: